front/public/locales/en/infraEditor.json

```diff
@@ -78,11 +78,11 @@
     }
   },
   "Electrification": {
-    "description": "A electrification is a set of cables used to power trains using a pantograph. The electrification is identified by an id",
+    "description": "An electrification is a set of cables used to power trains using a pantograph. The electrification is identified by an id",
     "properties": {
       "track_ranges": {
-        "description": "Liste des intervalles de TIV où la tension est appliquée.",
-        "title": "Intervalles de TIV"
+        "description": "Liste des intervalles de section de voie où la tension est appliquée.",
+        "title": "Intervalles de section de voie"
```

Comment:
Hm - is this untranslated? (This is in the English file)

Reply:
A lot of things haven't been translated yet, it could be a whole separate ticket.